webapp/lib/controller.dart

```diff
@@ -422,15 +422,24 @@ void initUI() {
       applyConfiguration(newConfig);
     }
   );
+  // Apply the default configuration before loading any new configs.
+  applyConfiguration(defaultUserConfig);
 }
 
 
 /// Sets user customization flags from the data map
 /// If a flag is not set in the data map, it defaults to the existing values
 void applyConfiguration(model.UserConfiguration newConfig) {
-  if (currentConfig.keyboardShortcutsEnabled != newConfig.keyboardShortcutsEnabled) {
-    newConfig.keyboardShortcutsEnabled ? view.replyPanelView.showShortcuts() : view.replyPanelView.hideShortcuts();
-    newConfig.keyboardShortcutsEnabled ? view.tagPanelView.showShortcuts() : view.tagPanelView.hideShortcuts();
+  if (currentConfig.repliesKeyboardShortcutsEnabled != newConfig.repliesKeyboardShortcutsEnabled) {
+    newConfig.repliesKeyboardShortcutsEnabled ? view.replyPanelView.showShortcuts() : view.replyPanelView.hideShortcuts();
+  }
+
+  if (currentConfig.tagsKeyboardShortcutsEnabled != newConfig.tagsKeyboardShortcutsEnabled) {
+    newConfig.tagsKeyboardShortcutsEnabled ? view.tagPanelView.showShortcuts() : view.tagPanelView.hideShortcuts();
+  }
+
+  if (currentConfig.sendMessagesEnabled != newConfig.sendMessagesEnabled) {
+    newConfig.sendMessagesEnabled ? view.replyPanelView.showButtons() : view.replyPanelView.hideButtons();
   }
```

Comment:
The view doesn't remember its show/hide state, but even so, I like your API design much better, I think it makes the code more straightforward, so I've now updated all of the methods to use it instead of the show/hide style. Thanks!